test/k8s_logging_tests/test_config.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import time
+import os
+import logging
+import json
+from urllib.parse import urlparse
+from ..common import check_events_from_splunk
+
+
+@pytest.mark.parametrize("test_input,expected", [
+    ("test_data", "")
```

Comment:
should test_data be empty as it isn't used anywhere

Reply:
It's "" here coz its checking for !="" in assert.

Reply:
the input to the functions is in "test_input,expected" pairs, so we should have input, output pairs we want to assert on. In this "test_data" is not really an input used anywhere inside the test. ("test_data", 0) pair makes sense to me when you assert >= 0(expected value in this case)

Reply:
have updated it to ("test_data", 1) not `0` coz it needs to have some data.